src/qt/sendcoinsdialog.cpp

```diff
@@ -383,17 +386,36 @@ void SendCoinsDialog::on_sendButton_clicked()
         return;
     }
 
-    // now send the prepared transaction
-    WalletModel::SendCoinsReturn sendStatus = model->sendCoins(currentTransaction);
-    // process sendStatus and on error generate message shown to user
-    processSendCoinsReturn(sendStatus);
+    bool done = false;
```

Comment:
`done` isn't really helping me parse what is going on here, maybe call it `send_failure` and reverse the meaning?